src/app/cube/details/splash/details-splash.component.html

```diff
@@ -6,7 +6,9 @@
         <span tabindex="0">Created by </span>
         <a tabindex="0" [routerLink]="['/users', learningObject.author.username]" class="link" id="author-link" attr.aria-label="Clickable link {{ learningObject.author.name }}">{{learningObject.author.name | titlecase}}</a> - 
         <a tabindex="-0" [routerLink]="['/organization', learningObject.author.organization]" class="link"  attr.aria-label="Clickable link {{ learningObject.author.organization }}">{{learningObject.author.organization | titlecase}}</a>
-        <span tabindex="0">, updated {{ learningObject.date | date:'shortDate' }}</span>
+        <div class="updated" tabindex="0">Updated {{ learningObject.date | date:'shortDate' }}
+          <a *ngIf="canViewChangelogs" id="view-changelog" (click)="openViewAllChangelogsModal()" aria-label="Clickable link view the Changelog" tabindex="0">View the changelog</a>
```

Comment:
`<a *ngIf="canViewChangelogs" id="view-changelog" (click)="openViewAllChangelogsModal()" attr.aria-label="View the changelog for {{ learningObject.name }}" tabindex="0">View the changelog</a>`